tools/cdata.js

```diff
@@ -267,7 +267,7 @@ writeChunks(
         str
           .replace(
             /function GetV().*\<\/script\>/gms,
-            "</script><script src=\"settings/settings.js?p=0\"></script>"
+            "</script><script src=\"/settings/s.js?p=0\"></script>"
```

Comment:
FYI: including / at the beginning may break reverse proxy configurations.

Reply:
Right! Will make sure they do not start with `/` although that requires the workaround with both .js/.css handlers as the main settings page will access `/settings.js` while sub pages access `/settings/settings.js`. Adding redundant handlers is the lesser problem compared to proxies not working though!